pkg/kubectl/cmd/apply_test.go

```diff
@@ -403,3 +405,87 @@ func testApplyMultipleObjects(t *testing.T, asList bool) {
 		t.Fatalf("unexpected output: %s\nexpected: %s OR %s", buf.String(), expectOne, expectTwo)
 	}
 }
+
+const (
+	filenameDeployObjServerside = "../../../test/fixtures/pkg/kubectl/cmd/apply/deploy-serverside.yaml"
+	filenameDeployObjClientside = "../../../test/fixtures/pkg/kubectl/cmd/apply/deploy-clientside.yaml"
+)
+
+func validateNULLPatchApplication(t *testing.T, req *http.Request) {
```

Comment:
Would you also add a test exactly the same, but for non-null patch.  e.g. the map does not contain the value.  This makes it easier to look at the tests and reason about the exact behavior.